exchange/order/limits/limits_types.go

```diff
@@ -70,6 +70,7 @@ type MinMaxLevel struct {
 	MarketStepIncrementSize float64
 	MaxTotalOrders          int64
 	MaxAlgoOrders           int64
+	Launch                  time.Time
```

Comment:
🚧 Change request:
`Listing`, `Listed`, `Active` all feel fine.
Suffixing `Time` feels good too.
```suggestion
	ListingTime                  time.Time
```

Reply:
Thanks; opted for `Listed` without suffix as we can take it from type 🤷 also dropped the setting of spot pairs `delisting` field because it's not really correct. I think there should be a time window only if the exchange supplies it explicitly. 

https://github.com/thrasher-corp/gocryptotrader/pull/2018/commits/f2a6291d993987e7f38343edfd63650a163ba26b